logdetective/server/database/models/merge_requests.py

```diff
@@ -0,0 +1,396 @@
+import datetime
+from typing import Optional
+
+import backoff
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    DateTime,
+    String,
+    ForeignKey,
+    UniqueConstraint,
+    desc,
+)
+from sqlalchemy.orm import relationship
+from sqlalchemy.exc import OperationalError
+from logdetective.server.database.base import Base, transaction, DB_MAX_RETRIES
+
+
+class MergeRequests(Base):
```

Comment:
It's possible over an extended lifetime of Log Detective that we will need to handle multiple Gitlab instances hosted in different places. We should probably include an instance ID here (and create another table that provides that instance's base URL, such as `https://gitlab.com` or `https://gitlab.example.com`.

This will also mean changing the uniqueness constraints to include this new field.